.github/workflows/integrate-enforce-docs/action.yaml

```diff
@@ -50,9 +50,18 @@ runs:
         rsync -av --delete --exclude="_index.md" \
           "/tmp/chainctl-docs/" "content/chainguard/chainguard-enforce/chainctl-docs/"
 
-    - name: fix chainctl-docs urls
```

Comment:
This fix task and the sed lines aren't used any more since chainctl generates the right URLs now